### PR TITLE
feat:Adding the project structure

### LINKED ISSUE #14: 📚 Documentation Enhancement
## Issue Overview

Our project documentation is the gateway for newcomers to understand, contribute, and make the most of our projects. To ensure it's informative, user-friendly, and up-to-date, we need your expertise!

### What's Needed

We're looking for contributors to help us enhance our project documentation in the following ways:

1. **Content Review**: Review the existing documentation for accuracy, clarity, and completeness.
2. **Updating**: Identify outdated information and update it to reflect the current state of the project.
3. **Clarity**: Simplify complex topics and ensure that even beginners can understand our documentation.
4. **Organization**: Suggest improvements in the structure and organization of the documentation.

### How You Can Contribute

1. **Fork the Repository**: Fork our project repository to work on the documentation enhancements.
2. **Make Your Changes**: Make the necessary updates and improvements to the documentation.
3. **Create a Pull Request**: Submit a pull request with your changes. Ensure that you follow our [Contribution Guidelines](https://github.com/mlsanigeria/speak-to-docs/blob/main/CONTRIBUTING.md) for detailed instructions.
4. **Review and Collaboration**: We'll review your pull request and collaborate to finalize the documentation enhancements.

### Let's Make Learning and Contributing Easier!

By contributing to our documentation, you're helping us create a better experience for everyone in our community. Your input will be invaluable for both newcomers and seasoned contributors.

**Thank you for making our project even better!** 🙌

**Note**: If you have any questions or need assistance, feel free to ask in the comments.

### PR DESCRIPTION
**Pull request by PetJs**

### CODE REVIEW

**Review by Sammybams — CHANGES_REQUESTED** [review]
Thank you very much @PetJs. We cannot merge your Pull request at this moment because you have used generic names. Some of the directories and files you have referenced do not exist in the repo. Please let you contributions align strictly with what is currently in the repo. We would be happy to merge your PR once you make these changes. Thank you very much.

**Comment by PetJs** [comment]
Can I refactor the file tree, making sure people know where to go when they want to work on some issues

**Comment by Sammybams** [comment]
> Can I refactor the file tree, making sure people know where to go when they want to work on some issues

Edit your contribution to reflect what is exactly in the repo at the moment.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,14 +72,15 @@ Welcome to the **Speak-To-Docs** project repository, organized by the Microsoft
 
 The project is structured as follows:
 - **speech_to_docs**: This is the main directory for the project.
-- **speech_to_docs/src/data**: This directory contains the dataset used for training the model.
-- **speech_to_docs/src/model**: This directory contains the model architecture and training code.
-- **speech_to_docs/src/utils**: This directory contains utility functions used throughout the project.
+- **speech_to_docs/src**: This directory contains all the files that will be used for training the model.
```
Comment (Sammybams) [inline CONTRIBUTING.md]:
```suggestion
- **speech_to_docs/src**: This directory contains all the files that will house all the functionalities of the project: Speech transcription and synthesis, RAG Model Solution and document reading.

```

**Review by Sammybams — APPROVED** [review]
Thank you @PetJs. Great work.

**Pull request merged**